Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -115,6 +115,21 @@ void JitArm64::reg_imm(u32 d, u32 a, u32 value, u32 (*do_op)(u32, u32),
   }
 }
 
+u32 bit_or(u32 a, u32 b)
```

Comment:
Good catch, fixed. Yeah I took a look at those functors, it's a shame they're not also provided as actual functions :/